go.mod

```diff
@@ -24,7 +24,7 @@ require (
 	github.com/ipfs/interface-go-ipfs-core v0.4.0
 	github.com/lazyledger/lazyledger-core/p2p/ipld/plugin v0.0.0-20210219190522-0eccfb24e2aa
 	github.com/lazyledger/nmt v0.2.0
-	github.com/lazyledger/rsmt2d v0.0.0-20201215203123-e5ec7910ddd4
+	github.com/lazyledger/rsmt2d v0.0.0-20210326165230-b6417926360b
```

Comment:
Orthogonal to this PR but we should start tagging releases for rsmt2d. I just tagged a pre-release here: https://github.com/lazyledger/rsmt2d/releases/tag/v0.1.0